src/app/view/perspectives/chart/chart-perspective.component.html

```diff
@@ -18,7 +18,7 @@
         [@slide]
         [attributes]="documentAttributes"
         [preSelected]="attributeY"
-        [emptyDescription]="'Y Axis'"
+        [emptyDescription]="'Y ' + axisTitle()"
```

Comment:
Maybe I would translate the whole thing separately for each axis as it might be translated differently in various languages.